js/app.js

```diff
@@ -171,6 +170,28 @@ function getSeatListHtml(seats) {
   return seatList;
 }
 
+function createMapImageOverlay(map) {
```

Comment:
Maybe "createMapImageOverlayHtml" or "getMapImageOverlayHtml", just to match other html-generating functions. We could even split view-focused functions out into a separate file at some point if we want to get real organized...but that sounds like work.

Reply:
Agree on matching the other functions. 

I also agree that we should have a separation between the logical "game state" operations (randomization, ordering, faction selection, potential further setup decisions from the new expansion...) and the render operation (which is really a function of "game state" -> "generated html"/model -> view). I don't think I want to introduce this in this PR but I do think we would benefit from this - and would be something I'd be down to play with in the near future. 

I also would like to suggest leveraging typing information to better encode constraints between these steps.

Reply:
Yay typing. I'm into it. Although it does somewhat depend on the end goal. It's maybe not worth the effort if we're not expanding the app very much. You have free time now so maybe that math changes a bit :)

Reply:
Renamed as discussed for now.